src/frontend/screens/Library/components/AlphabetFilter/index.css

```diff
@@ -0,0 +1,98 @@
+/*
+  Modern Alphabet Filter
+  - Uses CSS variables for easy theming.
+  - Aligned with a dark UI.
+  - Adds subtle animations and accessibility features.
+*/
+
+:root {
+  /* --- THEME COLORS (Adjust these to perfectly match your app) --- */
+  --filter-accent-color: #00aeef; /* A vibrant teal/cyan from your 'ADD GAME' button */
```

Comment:
we probably want to include some colors for the different themes in this PR, cause now all the themes would use  teal/cyan color when maybe that color is not the accent color of the theme